.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,34 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q            |   A
+|---------------- | ------
+| BC Break        | yes/no
+| Library Version | x.y.z <!-- provide patch version too -->
+| PHP version     | x.y.z
```

Comment:
does it make sense to indicate this for this type of product?

Reply:
I supposed that it could be useful but if you think that PHP version is not necessary I trust in you